concrete/src/Express/Entry/Manager.php

```diff
@@ -122,19 +123,18 @@ public function saveEntryAttributesForm(Form $form, Entry $entry)
     public function getEntryAttributeValuesForm(Form $form, Entry $entry)
     {
         $submittedAttributeValues = [];
+
         foreach ($form->getControls() as $control) {
             if($control instanceof AttributeKeyControl){
                 $attributeKey = $control->getAttributeKey();
-                $genericValue = $attributeKey->getController()->createAttributeValueFromRequest();
-                $attributeValue = new ExpressValue();
-                $attributeValue->setAttributeKey($attributeKey);
-                $attributeValue->setEntry($entry);
-                $attributeValue->setAttributeValueObject($genericValue);
+                $akHandle = $attributeKey->getAttributeKeyHandle();
```

Comment:
Hey, I'm not sure what this code is doing in the orphaned blocks commit. I totally missed it the first time. Is this an attempt to fix the duplicate attribute issue? I'm not sure the logic behind these changes but it looks pretty wrong to me. I'm reverting this.